content/vendor/guides/production-ready.md

```diff
@@ -53,10 +53,40 @@ If you expect to also install stateful services into existing clusters, you'll l
 
 ### Namespaces
 
-It is *strongly* advised that applications be architected to deploy a single application into a single namespace when possible. This will give the most flexibility when deploying to end user environments. Don't specify a namespace in your YAML resources, or try to make this user-configurable using the `kots.io` `Config` object, just leave namespace blank.
+It is *strongly* advised that applications be architected to deploy a single application into a single namespace when possible. This will give the most flexibility when deploying to end user environments. Most notably, it allows you to run with minimal Kubernetes permissions, which can reduce friction when an app runs as a tenant in a large cluster. Don't specify a namespace in your YAML resources, or try to make this user-configurable using the `kots.io` `Config` object, just leave namespace blank.
 
 Letting the end user manage namespaces will be the easiest way to reduce friction. The ability to manage namespaces in kotsadm will be added in an upcoming KOTS release, so if you intend to  deploy multiple apps using a single Admin Console, it is reasonable to architect your app assuming the end user will manage the Namespaces that each component runs in.
 
+
+```yaml
+# good, namespace absent
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: spline-reticulator
+spec:
+```
+
+```yaml
+# bad, hardcoded
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: spline-reticulator
+  namespace: graphviz-pro
+spec: 
+```
+
+```yaml
+# bad, configurable
```

Comment:
Saying this is bad seems to contradict the earlier opinion we state: “Letting the end user manage namespaces will be the easiest way to reduce friction...”
Maybe there’s a better way to word this? “Less good, unless configured through kots?”

Reply:
This is a good question. Let me take another pass and clarify what I mean by `Letting the end user manage namespaces`

Reply:
May be we can say "let the end user manage namespace and in your YAML add something like `namespace: repl{{ Namespace }}`' 🤷